### PR TITLE
Compatibility with NC12

### PR DESCRIPTION
**Pull request by raimund-schluessler**
This should fix all issues with NC12 so that master is compatible with both NC12 and NC13.

The file `css/src/nc12.scss` will be deleted once NC12 is EOL.

@skjnldsv Could you have a look and check that it works with NC12 and NC13, please?

### CODE REVIEW

**Comment by raimund-schluessler** [comment]
Ok, with this fix, the app is usable on NC12. There are some visual differences to the version on NC13, but only minor ones.

File: css/src/nc12.scss [hunk 1]
```diff
@@ -72,3 +72,205 @@
 	box-shadow: unset;
 	filter: drop-shadow(0 1px 3px rgba(179, 179, 179, .75));
 }
+
+$color-main-text: #000;
```
Comment (skjnldsv) [inline css/src/nc12.scss]: This is not needed, nc12 has these variables as well.
https://github.com/nextcloud/server/blob/stable12/core/css/variables.scss

File: css/src/nc12.scss [hunk 2]
```diff
@@ -72,3 +72,205 @@
...
+
+		.ui-icon {
+			opacity: .5;
+
+			&.ui-icon-circle-triangle-e {
+				background: url('../../../apps/tasks/css/../img/arrow-right.svg') center center no-repeat;
+			}
+
+			&.ui-icon-circle-triangle-w {
+				background: url('../../../apps/tasks/css/../img/arrow-left.svg') center center no-repeat;
```
Comment (skjnldsv) [inline css/src/nc12.scss]: This looks like a compiled path. use the correct url and let the scss compiler rewritter do its job :)

File: css/src/nc12.scss (hunk 1 above)
Comment (raimund-schluessler) [reply css/src/nc12.scss]: Thanks, I will remove it.

File: css/src/nc12.scss (hunk 2 above)
Comment (raimund-schluessler) [reply css/src/nc12.scss]: What would be the correct path? From the scss file to the svg file? Like `../img/arrow-left.svg`?

Comment (skjnldsv) [reply css/src/nc12.scss]: Yes, relative to the task app directory :)

**Comment by raimund-schluessler** [comment]
Fixed and rebased

**Pull request merged**